com.poweriso.PowerISO.json

```diff
@@ -9,7 +9,6 @@
         "--device=all",
         "--env=QT_QPA_PLATFORM_PLUGIN_PATH=/app/lib",
         "--filesystem=home",
-        /* Only supports X11 with the builtin Qt */
```

Comment:
It's unsupported by flatpak-external-data-checker. It's supported by Flatpak, and will work just fine. Please leave it as-is.